core/commands/resolve.go

```diff
@@ -52,9 +47,7 @@ Resolve the value of another name recursively:
 	Arguments: []cmds.Argument{
 		cmds.StringArg("name", true, false, "The name to resolve.").EnableStdin(),
 	},
-	Options: []cmds.Option{
-		cmds.BoolOption("recursive", "r", "Resolve until the result is an IPFS name"),
```

Comment:
why is this getting removed? we want to be able to resolve objects step by step. 

maybe `ipfs resolve <path>` should resolve all the way, but an option should exist to allow this.

also, this _changes_ cli UX and should be done carefully. it breaks userspace.